dart/lib/src/isolate_error_integration.dart

```diff
@@ -31,7 +31,7 @@ class IsolateErrorIntegration extends Integration {
 
 RawReceivePort _createPort(Hub hub, SentryOptions options) {
   return RawReceivePort(
-    (dynamic error) async {
+    (Object? error) async {
```

Comment:
I'd rather rollback those changes because of https://github.com/getsentry/sentry-dart/issues/511
so ideally we'd do it all together and go thru pre-releases to avoid errors

Reply:
I reverted all the dynamic changes except for the list types.